nemo_rl/models/policy/dtensor_policy_worker.py

```
@@ -1186,6 +1186,130 @@
+                    input_ids, position_ids, _ = pack_sequences(
+                        input_ids=input_ids,
+                        input_lengths=input_lengths,
+                        packed_sequence_size=[
+                            batch_size
+                        ],  # flash attention 2 expects flattened input
+                        padding_value=self.tokenizer.eos_token_id,
+                        return_attention_mask=False,
+                    )
+                    seq_len = input_ids.shape[1]
+                    attention_mask = None
+                    flash_attn_kwargs = get_flash_attention_kwargs(
```

Ruff (0.12.2)

Local variable `flash_attn_kwargs` is assigned to but never used

Remove assignment to unused variable `flash_attn_kwargs`

(F841)